### DISCUSSION TITLE
存在内存泄漏

### DISCUSSION THREAD

**Discussion by qingpingwang** [Tencent/libpag#2256] [General] [answered]
![image](https://github.com/Tencent/libpag/assets/50129609/8d7e2709-720d-4655-965a-532601568c66)
分支：release4.3
重复播放，内存占用从最初50MB，涨到500+MB

**Comment by kevingpqi123** [comment]
这类问题基于 pag 的官方demo 测试，如果存在这类问题，将复现方法或复现 demo 发出来，现在的反馈没有多少有效信息
如果基于官方 demo 测试没有问题，就排查自己的使用问题

**Reply by qingpingwang** [reply]
我想知道是什么问题，以及怎么去规避。比如是某些特效导致的。

**Reply by qingpingwang** [reply]
[test1.pag.zip](https://github.com/Tencent/libpag/files/15021251/test1.pag.zip)
mac案例替换成这个pag就能复现，测了其它的官方资源。好像都比较稳定。

**Reply by kevingpqi123** [reply]
如果仅仅只有这个素材有问题，是需要我们排查的，说明 pag sdk 在部分场景下有 OOM 的问题，是需要我们修复的

**Reply by qingpingwang** [reply]
好的，辛苦看一下。用的：https://github.com/Tencent/libpag/archive/refs/tags/v4.3.43.tar.gz

**Reply by kevingpqi123** [reply, marked as answer]
问题已经定位到，下周我们会发布一个版本修改该问题，感谢反馈

**Reply by qingpingwang** [reply]
请问Linux上会存在这个问题吗？我的主要使用场景是服务端。如果不影响我就不需要关注这个问题。

**Reply by kevingpqi123** [reply]
不会有，主要出现在 iOS 和 macOS 平台，且是特定类型的素材

**Reply by kevingpqi123** [reply]
4.3.56 版本已修复内存泄漏问题
